### DISCUSSION TITLE
Graphical system tools

### DISCUSSION THREAD

**Discussion by fadi123go** [SpiralLinux/SpiralLinux-project#500] [General]
Would you please create some graphical tools for spirallinux . similar to mx tools ,mageia os , or even  salix os.  They should include 
a user manager  , time sycnh tool. a service manager , codec installer, system information displayer . grub manager. driver manager and so on .

**Comment by geckolinux** [comment]
Hi there, nope, SpiralLinux does not create or maintain any software or packages. However, many of the items you mentioned are already included from what is available in Debian:

- user manager (pre-installed in all SpiralLinux editions, different depending on the desktop environment)
- time sync (pre-configured out-of-the-box and also manual time settings GUI in each desktop environment)
- service manager (pre-installed in the Plasma edition)
- codec installer (not needed, codecs are pre-installed in all SpiralLinux editions)
- system information (available in the settings of most desktop environments)
- grub manager ("Grub Customizer" pre-installed in all SpiralLinux editions)
- driver manager (most drivers are pre-installed in SpiralLinux)
